startup_scripts/200_vlan_groups.py

```diff
@@ -18,9 +18,17 @@
         if assoc in params:
             model, field = details
             query = {field: params.pop(assoc)}
-
-            params[assoc] = model.objects.get(**query)
-
+            # Get model from Contenttype
+            scope_type = params.pop("scope_type", None)
+            if not scope_type:
+                print("scope_type is missing from VLAN Group")
```

Comment:
Perhaps we could add the name of the current VLAN Group to the error message to ease debugging?